epub33/core/index.html

```diff
@@ -1191,6 +1217,11 @@ <h4>Package Document Definition</h4>
 					<p>All [[XML]] elements defined in this section are in the <code>http://www.idpf.org/2007/opf</code>
 						namespace [[XML-NAMES]] unless otherwise specified.</p>
 
+					<p>
+						To parse a URL string <var>url</var> used in the Package Document, the <a data-cite="url#concept-url-parser">URL Parser</a> [[URL]] MUST be used with the 
```

Comment:
Editorial nitpicking: the wording is not very usual wrt Infra. I would phrase it as I did in the comment. 

I assume you wanted to introduce a MUST here, but this is [covered by Infra](https://infra.spec.whatwg.org/#algorithm-conformance) (a imperative application of an algorithms implies a MUST).

Reply:
The reason I would like to keep _MUST_ is because it helps us to locate all places in the spec where tests are needed (and also provide anchors to refer to from tests). I understand that infra has this extra implicit rule, but it also does not prevent using _MUST_.

I have modified the sentence to say 

> `To parse a URL string <var>url</var> used in the Package Document, the <a data-cite="url#concept-url-parser">URL Parser</a> [[URL]] MUST be applied to <var>url</var>, with the <a>container root URL</a> as <var>base</var>.`

ie, instead of saying "apply" it now says "MUST be applied", otherwise it is the same as in the original comment. Would that work for you?

Reply:
OK, works for me 😊